### PR TITLE
documentation for importing data with docker

### PR DESCRIPTION
**Pull request by dippindots**
Fix https://github.com/cBioPortal/cbioportal/issues/7523

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ We also maintain an active [list of RFCs (Requests for Comments)](RFC-List.md) w
 
 ### 2.1.1 Deploy with Docker (Recommended)
 * [Deploy with Docker](docker/README.md)
+* [Import data with Docker](docker/import_data.md)
```
Comment (inodb) [inline docs/README.md:23]: good to also add to `SUMMARY.md`, this is what gitbook.io uses to make the table of contents (TOC). Ideally the TOC in `README.md` and `SUMMARY.md` are always the same. I should prolly add some test for that

File: docs/docker/README.md [hunk 2]
```diff
@@ -175,6 +175,7 @@ echo $LATEST_VERSION
 ```
 
 ## Data loading & more commands ##
+For importing sample study, see [this tutorial](import_data.md)
```
Comment (inodb) [inline docs/docker/README.md:178]:
```suggestion
For documentation on how to import a study, see [this tutorial](import_data.md)
```

File: docs/docker/import_data.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+## Import data instructions ##
+This is an example to import a sample study: `study_es_0`.
+When try to import other studies, please follow the same routine:
+1. import gene panels (if applicable, some studies may don't need to import gene panels)
+2. import data
```
Comment (inodb) [inline docs/docker/import_data.md:5]:
```suggestion
When trying to import other studies, please follow the same routine:
1. import gene panels (if applicable, studies without gene panels are assumed to be whole exome/genome)
2. import study data
```

File: docs/docker/import_data.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+### Step 1 - Import gene panels
+
+To import gene panels for your study, please referece the example commands in [this file](example_commands.md#importing-gene-panel)
```
Comment (inodb) [inline docs/docker/import_data.md:9]:
```suggestion
To import gene panels for your study, please reference the example commands in [this file](example_commands.md#importing-gene-panel)
```

File: docs/docker/import_data.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Commands for importing `study_es_0` gene panels(`data_gene_panel_testpanel1` and `data_gene_panel_testpanel2`):
```
Comment (inodb) [inline docs/docker/import_data.md:11]:
```suggestion
These are the commands for importing `study_es_0` gene panels (`data_gene_panel_testpanel1` and `data_gene_panel_testpanel2`):
```

File: docs/docker/import_data.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+docker run -it --rm \
+    --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    bash -c 'cd /cbioportal/core/src/main/scripts/ && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel2.txt'
+```
+
+### Step 2 - Import data
+
+To import data for your study, please referece the example commands in [this file](example_commands.md#importing-data)
```
Comment (inodb) [inline docs/docker/import_data.md:30]:
```suggestion
To import data for your study, please reference the example commands in [this file](example_commands.md#importing-data)
```

File: docs/docker/import_data.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    metaImport.py -u http://cbioportal-container:8080 -s /cbioportal/core/src/test/scripts/test_data/study_es_0
+```
+
+:warning: after importing a study, remember to restart `cbioportal-container`
+to see the study on the home page. Run `docker restart cbioportal-container`.
+
+
+## Common Questions and solutions
```
Comment (inodb) [inline docs/docker/import_data.md:45]:
```suggestion
## Frequently Asked Questions
```
I think this is the common title for these kind of sections

This is a great idea to include btw 👍

File: docs/docker/import_data.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+```shell
+docker run -it --rm --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    metaImport.py -u http://cbioportal-container:8080 -s /cbioportal/core/src/test/scripts/test_data/study_es_0
+```
+
+:warning: after importing a study, remember to restart `cbioportal-container`
+to see the study on the home page. Run `docker restart cbioportal-container`.
+
```
Comment (inodb) [inline docs/docker/import_data.md:43]: Maybe good to have some concluding notes at the end. E.g.:

You have now imported the test study `study_es_0`. This study is included inside the cbioportal container. The process for adding a study that is outside of the container is almost the same. The difference is that one needs to add the `-v` parameter to mount the folder with the study data inside the container. See `How to import a real study` section (add markdown link) for an example.

File: docs/docker/import_data.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+```shell
+docker restart cbioportal-container
+```
+
+### How to import a real study
+
+You can find all public studies at [cbioportal.org](http://www.cbioportal.org/data_sets.jsp) a zipped folder with staging files from each study can be downloaded. These zip files are compressed versions of the study folders in [datahub](https://github.com/cBioPortal/datahub).
+
+Take study `Cholangiocarcinoma (TCGA, PanCancer Atlas)` as an example:  
+Study `Cholangiocarcinoma (TCGA, PanCancer Atlas)` is a whole exome/genome study, so just need to download the study data and run the command from step2.
```
Comment (inodb) [inline docs/docker/import_data.md:66]:
```suggestion
You can find all public studies at [cbioportal.org](http://www.cbioportal.org/data_sets.jsp) a zipped folder with staging files from each study can be downloaded. These zip files are compressed versions of the study folders in [datahub](https://github.com/cBioPortal/datahub).

Download and extract the zip file for study `Cholangiocarcinoma (TCGA, PanCancer Atlas)`. Study `Cholangiocarcinoma (TCGA, PanCancer Atlas)` is a whole exome study. You can therefore skip step 1 from the import data instructions section (add markdown link): the default gene panel is whole exome. so there is no need to import a custom one. Proceed with the command from step2
```

**Comment by dippindots** [comment]
Thanks @inodb , updated!

**Pull request merged**